src/sql/workbench/browser/modelComponents/declarativeTable.component.ts

```diff
@@ -70,14 +71,19 @@ export default class DeclarativeTableComponent extends ContainerBase<any> implem
 		return !column.isReadOnly;
 	}
 
-	private isLabel(colIdx: number): boolean {
+	private isLabel(colIdx: number, r?: number): boolean {
```

Comment:
what does this `r` do?

Reply:
r is the rowIdx, good catch I'll change that.

Reply:
Huh i dont even use it, will remove it then